File: DevExpress-Demo/VB/DialogsDemo.Wpf/ViewModels/ThemedWindowModule/ContentModels/BaseThemedWindowContentModel.vb
Imports DevExpress.Mvvm.POCO

Namespace DialogsDemo

    Public MustInherit Class BaseThemedWindowContentModel

        Public ReadOnly Property RootViewModel As ThemedWindowViewModel
            Get
                Return GetParentViewModel(Of ThemedWindowViewModel)()
            End Get
        End Property
    End Class
End Namespace

File: DevExpress-Demo/CS/DialogsDemo.Wpf/ViewModels/ThemedWindowModule/ContentModels/BaseThemedWindowContentModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Mvvm.POCO;

namespace DialogsDemo {
    public abstract class BaseThemedWindowContentModel {
        public ThemedWindowViewModel RootViewModel { get { return this.GetParentViewModel<ThemedWindowViewModel>(); }}
    }
}
